purview/models/koji.py

```diff
@@ -0,0 +1,55 @@
+from neomodel import (
+    StructuredNode, StringProperty, IntegerProperty, UniqueIdProperty,
+    BooleanProperty, DateTimeProperty, FloatProperty, RelationshipTo)
+
+
+class KojiBuild(StructuredNode):
+    id_ = UniqueIdProperty(db_property='id')
+    tasks = RelationshipTo('KojiTask', 'TRIGGERS')
+    creation_event_id = IntegerProperty(required=True)
+    volume_id = IntegerProperty()
+    owner_id = IntegerProperty()
```

Comment:
This would be a good candidate for a separate node. We should have a generic user node (used for relationships with all labels) that has a relationship with the build.